examples/react-native/src/features/InAppMessaging/Demo/Example.tsx

```diff
@@ -6,9 +6,16 @@ import { SafeAreaView, StyleSheet, Text, View } from 'react-native';
 import { withInAppMessaging } from '@aws-amplify/ui-react-native';
 import { Button, Checkbox, Radio, RadioGroup } from '../../../ui';
 import { useInAppDemo, ACTIONS, LAYOUTS, ORIENTATIONS } from './utils';
-import config from './aws-exports';
+import { VERSION } from '@env';
 
-Amplify.configure(config);
+const AMPLIFY_CONFIG_PATH =
+  VERSION === 'gen1' ? 'src/amplifyconfiguration' : 'amplify_outputs';
+
+const amplifyOutputs = require(
```

Comment:
Have to use `require()` for dynamic import because `import()` is only supported by react native bundler when used  in conjunction with `React.lazy()` to load a lazy component.

Reply:
Note: we cannot extract `const AMPLIFY_CONFIG_PATH` and reuse it elsewhere because that will cause trouble for the bundler to do import analysis. It has to stay along with the `require` statement below.